CHANGELOG.md

```diff
@@ -2,6 +2,15 @@
 
 Reactist follows [semantic versioning](https://semver.org/) and doesn't introduce breaking changes (API-wise) in minor or patch releases. However, the appearance of a component might change in a minor or patch release so keep an eye on redesigns and make sure your app still looks and feels like you expect it.
 
+# next
+
+-   [BREAKING] Renames the CSS variable for primary text content inside toasts and tooltip. Previous name was `--reactist-content-toast`, now replaced with the new name `--reactist-toast-content-primary`.
+-   [Feat] New CSS variable to control the color of secondary text inside toasts and tooltip: `--reactist-toast-content-secondary`.
+-   [BREAKING] The `Tooltip` component no longer accepts generic props to be forwarded as generic HTML attributes to the tooltip content container. In particular, it no longer accepts a `className` to customize its appearance.
```

Comment:
💭 From what you've seen in todoist-web and twist-web, does it look like there are occurrences that will have to be updated (i.e. stop passing `className`)?

Reply:
I have found some in twist-web though. Maybe I'll add a `exceptionallySetClassName` backdoor for now, to make future upgrades easier.

Reply:
I just checked (since you asked) and found three:

Todoist:
- https://github.com/Doist/todoist-web/blob/main/src/react_components/empty-state/empty-state-tooltip.tsx#L17

Twist:
- https://github.com/Doist/twist-web/blob/main/src/components/markdown-content/renderers/link-renderer-channel.tsx#L35
- https://github.com/Doist/twist-web/blob/main/src/components/markdown-content/renderers/link-renderer-thread.tsx#L53

Reply:
Yeah, those are a good find (I'm talking about the ones you found in Todoist). Especially because it's the kind of case where I would simply remove the custom class name, or at least check that I can do it without much consequences. A custom border and shadow on a tooltip is something we should not keep. So my thinking when being aggressive and removing the capabality to pass custom class names is that we could even remove our current instances where we do it.

Anyway, I added the `exceptionallySetClassName` backdoor for now. Which still does not mean I'll keep the current instances where we do this. I'll still try to get rid of as many cases as I can.